app/src/main/res/values-en/strings.xml

```diff
@@ -835,30 +835,30 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_crossing_type_marked">Marked</string>
     <string name="quest_crossing_type_unmarked">Without road markings</string>
 
-    <string name="quest_cycleway_title2">"Is there a cycleway here? What type?"</string>
-    <string name="quest_cycleway_resurvey_title">"Is this still the cycleway situation here?"</string>
-    <string name="quest_cycleway_value_track">cycle track</string>
-    <string name="quest_cycleway_value_lane">cycle lane</string>
-    <string name="quest_cycleway_value_lane_soft">cycle lane (dashed markings)</string>
+    <string name="quest_cycleway_title2">"Is there a bike lane or path here? What type?"</string>
+    <string name="quest_cycleway_resurvey_title">"Is this still the cycling situation here?"</string>
+    <string name="quest_cycleway_value_track">separated bike path</string>
+    <string name="quest_cycleway_value_lane">bike lane</string>
+    <string name="quest_cycleway_value_lane_soft">bike lane (dashed markings)</string>
     <string name="quest_cycleway_value_none">none</string>
     <string name="quest_cycleway_value_shared">lane shared explicitly with other traffic</string>
     <string name="quest_cycleway_value_suggestion_lane">bicycle suggestion lane</string>
     <string name="quest_cycleway_value_bus_lane">on bus lane</string>
-    <string name="quest_cycleway_value_separate">displayed separately on map</string>
+    <string name="quest_cycleway_value_separate">displayed separately on the map</string>
     <string name="quest_cycleway_value_sidewalk">explicitly shared sidewalk</string>
-    <string name="quest_cycleway_value_lane_dual">cycle lane, both directions</string>
-    <string name="quest_cycleway_value_track_dual">cycle track, both directions</string>
+    <string name="quest_cycleway_value_lane_dual">bike lane, both directions</string>
+    <string name="quest_cycleway_value_track_dual">separated bike path, both directions</string>
     <string name="quest_cycleway_value_none_but_no_oneway">none, but cyclists may use road in both directions</string>
     <string name="quest_cycleway_value_none_and_oneway">none, no cycling allowed in this direction</string>
-    <string name="quest_cycleway_answer_contraflow_cycleway">Also cycleway on other side…</string>
-    <string name="quest_cycleway_answer_no_bicycle_infrastructure">No cycleway at all</string>
+    <string name="quest_cycleway_answer_contraflow_cycleway">Also bike path on the other side…</string>
```

Comment:
```suggestion
    <string name="quest_cycleway_answer_contraflow_cycleway">Also a bike path on the other side…</string>
```

Reply:
What is the context of this string?  It's a sentence fragment and doesn't make much sense on its own.

Reply:
In oneways, the UI only prompts the user to specify the bike path situation on the right side. The user may tap on "Other answers..." -> "Also cycleway on other side…" to switch the UI so that he can answer the cycleway situation on both sides. For example, to specify that there is a cycle track on the left side in contra-flow direction.

However, since this is basically a button text, the text should be as short as possible, hence I left out the "a". So I'll resolve this.